crates/sui-framework/src/natives/crypto.rs

```diff
@@ -40,3 +40,38 @@ pub fn ecrecover(
         Err(_) => Ok(NativeResult::err(cost, INVALID_SIGNATURE)),
     }
 }
+
+pub fn bls12381_verify(
+    context: &mut NativeContext,
```

Comment:
@punwai please make an issue for calibrating the cost of this native and assign to me

Reply:
Will do, thanks¡